editor/src/clj/util/num.clj

```diff
@@ -177,3 +201,8 @@
 
 (definline normalized->uint [num]
   `(uint (normalized->uint-double ~num)))
+
+(definline normalized->long [num]
+  (if *unchecked-math*
```

Comment:
When we `(set! *unchecked-math* :warn-on-boxed)`, does it also mean we make math ops unchecked? Looks like we are, huh:
```clj
(add-doc-and-meta *unchecked-math*
  "While bound to true, compilations of +, -, *, inc, dec and the
  coercions will be done without overflow checks. While bound
  to :warn-on-boxed, same behavior as true, and a warning is emitted
  when compilation uses boxed math. Default: false."
  {:added "1.3"})
```
I always assumed we use `(set! *unchecked-math* :warn-on-boxed)` only to avoid boxing, but it looks like it also turns off overflow checking... Not sure if we ever want this?

Reply:
I think we want overflow checks disabled for most of what we're doing, honestly. I feel like bugs resulting from overflow are rare, and we want the code to be as performant as possible. I'm quite happy to give up overflow checks to know where boxed math is happening.